spring-web/src/test/java/akka/stream/alpakka/spring/web/MyAppConfiguration.java

```diff
@@ -1,42 +0,0 @@
-/*
```

Comment:
instead I made it "autoconfig" which is a magical thing they exec when they load classpath stuffs